slik_wrangler/preprocessing.py

```diff
@@ -546,7 +546,7 @@ def get_attributes(data=None,target_column=None):
         cat_attributes.remove(target_column)
     else:
         pass
-    return num_attributes, cat_attributes
+    return {"Numerical Features": num_attributes, "Categorical Features": cat_attributes}
```

Comment:
@Justus-coded Why did you make it a dictionary? Also the get_attributes fn is used in a couple of places and can cause some functions to break. cc @AdesholaAfolabi

Reply:
My bad, didn't think about how my changes could affect the other places the function was used. 

The get_attr func returns both lists for num and cat features. The output from streamlit wasn't descriptive, that's why i used a dictionary.

Reply:
Yeah I saw. Can you revert to the previous return statement and handle the appearance on the streamlit script. You can return both objects and store in a dictionary instead. This should help right?

Reply:
Yes, it should. 

Thank you.